### PR TITLE
two models, with and without gender

### PR DESCRIPTION
**Pull request by ginoemiliozzi**
Quedaron 2 modelos, uno que usa el genero como input ademas de la imagen y el otro que solo usa la imagen.
Agregue un callback para reducir el learning rate cuando pasan 3 epocas sin notar mejoria en la funcion val_loss para evitar que de saltos bruscos al intentar minimizarla.
Agregue una capa dense de 240 que en la prueba con 500 imagenes me dio buen resultado, pero hay que ver si realmente funciona con mas imagenes.

### CODE REVIEW

**Comment by pablobots** [comment]
Buenísimo Gino, les comento como comprendí que queríamos hacer.


Para este papaer usaremos sólo el modelo que NO tiene el genero.


Osea el modelo ni se entera que existe la variable género.


Con el modelo sin género:

  *   Entrenamos con todas las imágenes y después obtenemos el MAE sólo para las imágenes de manos de hombres por un lado y el MAE sólo para las manos de mujeres por el otro.
  *   Entrenamos sólo con manos de hombres y obtenemos el MAE sólo para las imágenes de manos de hombres.
  *   Entrenamos sólo con manos de mujeres y obtenemos el MAE sólo para las imágenes de manos de mujeres.

Después comparamos los MAEs.

¿Les parece correcto?

Saludos.


________________________________
De: Gino Emiliozzi <notifications@github.com>
Enviado: miércoles, 1 de agosto de 2018 02:38 a. m.
Para: deeplearningrosario/deeplearningforcomputervision
CC: Subscribed
Asunto: [deeplearningrosario/deeplearningforcomputervision] two models, with and without gender (#18)


Quedaron 2 modelos, uno que usa el genero como input ademas de la imagen y el otro que solo usa la imagen.
Agregue un callback para reducir el learning rate cuando pasan 3 epocas sin notar mejoria en la funcion val_loss para evitar que de saltos bruscos al intentar minimizarla.
Agregue una capa dense de 240 que en la prueba con 500 imagenes me dio buen resultado, pero hay que ver si realmente funciona con mas imagenes.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18

[https://avatars1.githubusercontent.com/u/22394483?s=400&v=4]<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18>

two models, with and without gender by ginoemiliozzi · Pull Request #18 · deeplearningrosario/deeplearningforcomputervision<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18>
github.com
Quedaron 2 modelos, uno que usa el genero como input ademas de la imagen y el otro que solo usa la imagen. Agregue un callback para reducir el learning rate cuando pasan 3 epocas sin notar mejoria ...

Commit Summary

  *   two models, with and without gender

File Changes

  *   A .idea/deeplearningforcomputervision.iml<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-0> (11)
  *   A .idea/misc.xml<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-1> (4)
  *   A .idea/modules.xml<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-2> (8)
  *   A .idea/vcs.xml<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-3> (6)
  *   A .idea/workspace.xml<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-4> (282)
  *   M check.py<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-5> (2)
  *   A check_no_gender.py<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-6> (114)
  *   M main.py<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-7> (28)
  *   A no_gender.py<https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18/files#diff-8> (199)

Patch Links:

  *   https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18.patch
  *   https://github.com/deeplearningrosario/deeplearningforcomputervision/pull/18.diff

**Comment by sdsantiagodiez** [comment]
Coincido 👍
Muy bueno lo que salió de la reunión

**Comment by MaraniMatias** [comment]
¿Que sos esos archivos ` .idea/*.iml` son de tu IDE ?
Si son del IDE, Agregalos al `.gitignore ` y borrarlos de Pull-Requests :D

**Comment by ginoemiliozzi** [comment]
Listo mati, ahi los borre ! Perdon jaja
